### PR TITLE
Docs 5527 update rtm intro

### PR DESCRIPTION
**Pull request by hannanelson**
PDF:
[14-after-Anypoint Runtime Manager _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-runtime-manager/files/4713038/14-after-Anypoint.Runtime.Manager._.MuleSoft.Documentation.pdf)

### CODE REVIEW

**Review by mcocaro — APPROVED** [review]
LGTM - not fully convinced deployment strategies is the right term here as we usually use that term for blue/green, rolling deployments etc. which are within a deployment option.

**Comment by hannanelson** [comment]
> LGTM - not fully convinced deployment strategies is the right term here as we usually use that term for blue/green, rolling deployments etc. which are within a deployment option.

@mcocaro - I don't think we mention strategies here - it's still in the filename of the Deployment Options topic, though.

**Review by eketabch — APPROVED** [review]
EK

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .xref:index.adoc[Runtime Manager]
-* xref:index.adoc[About Runtime Manager]
-* xref:runtime-manager-switch-env.adoc[Using Different Environments in Runtime Manager]
-* xref:deployment-strategies.adoc[Deployment Options]
- ** xref:deploying-to-cloudhub.adoc[Deploying to CloudHub]
- ** xref:deploying-to-your-own-servers.adoc[Deploying to Your Servers]
+* xref:index.adoc[Anypoint Runtime Manager]
+* xref:runtime-manager-switch-env.adoc[Switch Environments using Runtime Manager]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc]:
```suggestion
* xref:runtime-manager-switch-env.adoc[Switch Environments Using Runtime Manager]
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,28 +1,60 @@
...
+
+With Runtime Manager, you can deploy, manage, and monitor your Mule applications in a sandbox, staging,
+or production environment from one central location, whether your apps are deployed in the cloud or on-premises.
+
+You can use Runtime Manager in these scenarios: 
+
+CloudHub::
+MuleSoft hosts your app deployments in CloudHub and you use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
++
+CloudHub is a complete integration platform as a service (iPaaS) that covers all of your server needs out of the box, including object store infrastructure, monitoring, troubleshooting, and scheduling.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion
CloudHub is a complete integration platform as a service (iPaaS) that automatically meets all of your server needs without requiring any configuration or customization, including object store infrastructure, monitoring, troubleshooting, and scheduling.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,28 +1,60 @@
...
 
-== Architecture
+Hybrid::
+You host your app deployments on your on-premises Mule servers and use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: Is this correct? parallelism
```suggestion
Your on-premises Mule servers host your app deployments and you use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,28 +1,60 @@
...
 
-The cloud-based version of the Anypoint Platform has a Runtime Manager with a console that allows you to manage applications deployed to xref:cloudhub.adoc[CloudHub] or elsewhere. 
+Anypoint Platform Private Cloud Edition::
+You host your app deployments on your local Mule servers and use the on-premises Runtime Manager instance
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: ditto re. parallel with first list item:
```suggestion
Your local Mule servers host your app deployments and you use the on-premises Runtime Manager instance 
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,28 +1,60 @@
...
-The cloud-based version of the Anypoint Platform has a Runtime Manager with a console that allows you to manage applications deployed to xref:cloudhub.adoc[CloudHub] or elsewhere. 
+Anypoint Platform Private Cloud Edition::
+You host your app deployments on your local Mule servers and use the on-premises Runtime Manager instance 
+included with Anypoint Platform PCE to manage them.
++
+If you are using the on-premises Runtime Manager, you can deploy anywhere _except_ CloudHub.
 
-If you are using the on-premises Runtime Manager you can deploy anywhere _except_ CloudHub.
+Anypoint Runtime Fabric::
+You host your app deployments on your locally installed Runtime Fabric instance and use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: ditto; reword like the others

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -1,28 +1,60 @@
...
 
-For applications deployed to CloudHub (Cloud runtime in the diagram below) you can use xref:insight.adoc[Insight] for analytics and monitoring. For applications that you deploy on-premises or to your private cloud servers you can xref:sending-data-from-arm-to-external-analytics-software.adoc[send data to third-party Analytics applications].
+// image::arm-big-picture.png[]
+image::rtm-overview.png[]
 
-image::arm-big-picture.png[]
+For applications deployed to CloudHub, you can use Insight for analytics and monitoring.
 
-Runtime Manager provides a dashboard that indicates the status of every deployed app and monitored server, with a rich set of features you can use to manage and track activity of your running applications.
+For applications that you deploy on-premises or to your private cloud servers, you can send data to third-party analytics applications.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]: Would there be any advantage to your audience to have the analytics info in each of the above descriptions?

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -1,28 +1,60 @@
...
+[NOTE]
+When you run an app in Anypoint Studio, the app deploys to an embedded test server in Studio.
+Because this server is not meant for production deployment and uptime restrictions apply,
+deploy your Mule app using one of the deployment options supported by Runtime Manager.
 
 == See Also
 
-* xref:runtime-fabric::deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric]
\ No newline at end of file
+* xref:cloudhub.adoc[CloudHub]
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:57]: I hope we can expand this title at some point....

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -1,28 +1,60 @@
...
+* xref:private-cloud::index.adoc[Anypoint Platform Private Cloud Edition]
+* xref:runtime-fabric::index.adoc[Anypoint Runtime Fabric]
+* xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
+* xref:deploying-to-your-own-servers.adoc[Deploy to Your Servers]
+* xref:deployment-strategies.adoc[Deployment Options]
+* xref:insight.adoc[Insight]
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:63]: ditto

**Review by kbostrom — APPROVED** [review]
Edited

**Pull request merged**